composer.json

```
@@ -1,15 +1,32 @@
 {
   "name": "mautic/helloworld-bundle",
-  "description": "Hello World plugin built on the Integrations framework.",
+  "description": "Hello World plugin built on the Integrations.",
   "type": "mautic-plugin",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "license": "GPL-3.0-or-later",
   "keywords": ["mautic","plugin","integration"],
+  "minimum-stability": "dev",
   "require": {
-    "php": ">=7.4.0 <8.1",
-    "mautic/core-lib": "^4.0"
+    "php": ">=8.0.0",
```

```suggestion
    "php": ">=8.1.0",
```
See: https://mautic.org/mautic-requirements/
M6 supports PHP 8.1, 8.2 and 8.3.